cognite/powerops/client/shop/cogshop_api.py

```diff
@@ -36,11 +55,364 @@ def auth(r: requests.PreparedRequest) -> requests.PreparedRequest:
         )
         response.raise_for_status()
 
+    def prepare_shop_scenario(

```

Comment:
Otherwise I think we should add optional arguments  `model_file, base_attribute_mappings, attribute_mapping_overrides, commands` etc.

Reply:
It was only used for a small part of the advanced guide. I will take it out of `cogshop_api.py` and instead use the generated sdk's object directly. 

Otherwise this file just becomes manually written wrappers for the already generated sdk